benchmark/pkg/runselector/schemes/last_n_runs.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package schemes
+
+import (
+	"k8s.io/contrib/test-utils/utils"
+)
+
+// Returns a list of run numbers of last 'n' successfully completed runs of
+// 'job'. If it cannot find n runs, returns as many runs as it could collect.
```

Comment:
I don't think that's a good behavior. It basically means that when the test starts to fail, we won't be able to use this. It should just take last n finished runs.